Tools-Override/tests.targets

```diff
@@ -47,7 +47,8 @@
   <PropertyGroup>
     <XunitResultsFileName>testResults.xml</XunitResultsFileName>
 
-    <XunitOptions Condition="'$(BuildingNETFxVertical)' == 'true'">$(XunitOptions) -noshadow -noappdomain </XunitOptions>
+    <XunitOptions Condition="'$(BuildingNETFxVertical)' == 'true'">$(XunitOptions) -noshadow</XunitOptions>
```

Comment:
No, actually the -noappdomain was causing other tests (multiple) to fail. Actually it doesn't hurt the tests executions it hurts the assert failure logs. If the tests are running in multiple app domains e.g `System.ComponentModel.EventBasedAsync` and an assert fails it would throw a `SerializableException` instead of the actual assert failure. I will try to identify other projects that have this problem and enable adding -noappdomain for them.